Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALTimeCalibData.h

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \class EMCALTimeCalibData
+/// \brief  Perform the EMCAL time calibration
+/// \author Joshua Koenig
+/// \ingroup EMCALCalib
+/// \since Jul 25, 2021
+
+#ifndef EMCAL_TIME_DATA_H_
+#define EMCAL_TIME_DATA_H_
+
+#include "DetectorsCalibration/TimeSlotCalibration.h"
+#include "DetectorsCalibration/TimeSlot.h"
+#include "DataFormatsEMCAL/Cell.h"
+#include "EMCALBase/Geometry.h"
+#include "CCDB/CcdbObjectInfo.h"
+
+#include "Framework/Logger.h"
+#include "CommonUtils/MemFileHelper.h"
+#include "CCDB/CcdbApi.h"
+#include "DetectorsCalibration/Utils.h"
+#include <boost/histogram.hpp>
+#include <boost/histogram/ostream.hpp>
+#include <boost/format.hpp>
+
+#include <array>
+#include <boost/histogram.hpp>
+namespace o2
+{
+namespace emcal
+{
+
+// holds the time calibration parameters
+class TimeCalibParams
+{
+  std::array<float, 17776> arr;
+};
```

Comment:
Can be using directives inside class:

``` c++
using TimeCalibParams = std::array<float, 17776> arr;
```

Actually the object is not needed because the process function should return directly o2::emcal::TimeCalibrationParams (Detectors/EMCAL/calib), so it can be removed entirely.

Reply:
deleted the TimeCalibParams and now use the dedicated TimeCalibrationParams